README.md

```diff
@@ -126,6 +126,37 @@ jobs:
         flox nix copy --to "$FLOX_SUBSTITUTER" -v nixpkgs#hello
 ```
 
+## Use an alternative `flox` executable
```

Comment:
For clarity I think you could give a little bit more information about what you mean by URI in this context. Do we list anywhere what the expected/accepted URIs look like? When I hear "URI" I think "URL" (whether I'm uninformed or not is besides the point), so something like `github:/flox/floxpkgs` makes sense, but something like `.#flox` breaks the mental model.

Reply:
Good point. I've added inline links to existing documentation about the URI syntaxes referenced here.